src/infi/systray/win32_adapter.py

```diff
@@ -138,16 +147,19 @@ class NOTIFYICONDATA(ctypes.Structure):
                 ("uFlags", ctypes.c_uint),
                 ("uCallbackMessage", ctypes.c_uint),
                 ("hIcon", HANDLE),
-                ("szTip", ctypes.c_char * 64),
+                ("szTip", ctypes.c_char * SZTIP_MAX_LENGTH),
                 ("dwState", ctypes.c_uint),
                 ("dwStateMask", ctypes.c_uint),
                 ("szInfo", ctypes.c_char * 256),
                 ("uTimeout", ctypes.c_uint),
+                # UINT  uVersion;  // used with NIM_SETVERSION, values 0, 3, 4

```

Comment:
uTimeout and uVersion are part of a union (meaning they are overloaded - they occupy the same 4-byte space and are used as either-this-one-or-that-one depending on context). I don't think it's necessary to specify both for our purposes - specifying just one is good enough.